tests/suite/fixtures.py

```diff
@@ -670,6 +671,140 @@ def fin():
     request.addfinalizer(fin)
 
 
+@pytest.fixture(scope="class")
+def crd_ingress_controller_with_dos(
```

Comment:
this function `crd_ingress_controller_with_dos` is very similar to `crd_ingress_controller_with_ap` and `crd_ingress_controller` - they all deploy and delete IC CRDs (vs,ts,gc...)

would it make sense to have a separate fixture that is responsible for that? `crd_ingress_controller_with_dos`, `crd_ingress_controller_with_ap` and `crd_ingress_controller` can all depend on it. that will reduce the duplication